src/styles/view-switcher.scss

```diff
@@ -0,0 +1,62 @@
+.view-switcher {
+  display: flex;
+  justify-content: center;
+
+  // These are the exact margins required to vertically align the top of the
+  // view switcher with the top of the Hide Highlights button to its left,
+  // and the top of the first annotation card below the view switcher with the
+  // top of the New Page Note button to its left.
```

Comment:
These comments are very helpful. If we can find a way to do the alignment "in code" via shared variables etc. that's even better.